test/integration/deployitems/timeout.go

```diff
@@ -0,0 +1,107 @@
+// SPDX-FileCopyrightText: 2021 SAP SE or an SAP affiliate company and Gardener contributors.
```

Comment:
couldn't we test that also as unit test with the envtest?

Reply:
see above